core/library.py

```diff
@@ -157,7 +158,7 @@ def get_movies(url):
                 else:
                     movie['resolution'] = 'DVD-SD'
             else:
-                movie['videocodec'] = movie['resolution'] = ''
+                movie['videocodec'] = ''
```

Comment:
If you remove this the 'resolution' key will not exist if the metadata cannot be parsed by hachoir. This key should be assigned here so that it is a blank string rather than null in the database.

Reply:
“Unknown” is even more accurate but I wanted to respect the pattern and further more Unknown-SD is a used else where so integration was closer to as-is.